packages/foreman/nodejs-jquery-ui/nodejs-jquery-ui.spec

```diff
@@ -0,0 +1,45 @@
+%global npm_name jquery-ui
+%global enable_tests 1
+
+Name: nodejs-%{npm_name}
+Version: 1.12.1
+Release: 1%{?dist}
+Summary: A curated set of user interface interactions, effects, widgets, and themes built on top of the jQuery JavaScript Library
+License: MIT
+Group: Development/Libraries
+URL: http://jqueryui.com
+Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+BuildRequires: nodejs-packaging
+BuildArch: noarch
+ExclusiveArch: %{nodejs_arches} noarch
+
+%description
+%{summary}
+
+%prep
+%setup -q -n package
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr external %{buildroot}%{nodejs_sitelib}/%{npm_name}
```

Comment:
This is actually very ugly and includes many bundled dependencies. Not sure how to get rid of this.